index.html

```diff
@@ -1,109 +1,130 @@
+        } else {
+          document.getElementById('wrap').style.visibility = 'inherit';
+        }
+      });
+    </script>
+  </head>
+  <body>
+    <a href="#container" class="skip-nav">본문 바로가기</a>
+    <div id="wrap">
+      <header class="header"></header>
+      <div id="container" class="main-page">
+        <section class="main-banner main-banner01">
+          <div class="inner02">
+            <div class="main-banner__cont">
+              <strong>
+                일상의 모든 물건을<br class="pc-only" />
+                거래해 보세요
+              </strong>
+              <a href="items.html">구경하러 가기</a>
+            </div>
           </div>
-        </a>
-      </div>
-    </section>
-    <section class="main-common main-search">
-      <div class="inner">
-        <a href="#">
-          <figure><img src="images/main_search.png" alt="상품 검색 이미지"></figure>
-          <div>
-            <h2>Search</h2>
-            <b>구매를 원하는<br class="pc-only">상품을 검색하세요</b>
-            <p>
-              구매하고 싶은 물품은 검색해서<br>
-              쉽게 찾아보세요
-            </p>
+        </section>
+        <section class="main-common main-hot">
+          <div class="inner">
+            <a href="#">
+              <figure><img src="images/main/main_hot.png" alt="인기 상품 이미지" /></figure>
+              <div>
+                <h2>Hot item</h2>
+                <b>인기 상품을<br class="pc-only" />확인해 보세요</b>
+                <p>
+                  가장 HOT한 중고거래 물품을<br />
+                  판다 마켓에서 확인해 보세요
+                </p>
+              </div>
+            </a>
           </div>
-        </a>
-      </div>
-    </section>
-    <section class="main-common main-register">
-      <div class="inner">
-        <a href="#">
-          <figure><img src="images/main_register.png" alt="인기 상품 이미지"></figure>
-          <div>
-            <h2>Register</h2>
-            <b>판매를 원하는<br class="pc-only">상품을 등록하세요</b>
-            <p>
-              어떤 물건이든 판매하고 싶은 상품을<br>
-              쉽게 등록하세요
-            </p>
+        </section>
+        <section class="main-common main-search">
+          <div class="inner">
+            <a href="#">
+              <figure><img src="images/main/main_search.png" alt="상품 검색 이미지" /></figure>
+              <div>
+                <h2>Search</h2>
+                <b>구매를 원하는<br class="pc-only" />상품을 검색하세요</b>
+                <p>
+                  구매하고 싶은 물품은 검색해서<br />
+                  쉽게 찾아보세요
+                </p>
+              </div>
+            </a>
           </div>
-        </a>
-      </div>
-    </section>
-    <section class="main-banner main-banner02">
-      <div class="inner02">
-        <div class="main-banner__cont">
-          <strong>
-            믿을 수 있는<br>
-            판다마켓 중고 거래
-          </strong>
-        </div>
+        </section>
+        <section class="main-common main-register">
+          <div class="inner">
+            <a href="#">
+              <figure><img src="images/main/main_register.png" alt="인기 상품 이미지" /></figure>
```

Comment:
💊 제안
제 생각에 figure는 독립적인 이미지에 사용하는 태그라 그냥 img 태그 사용하시는 것 추천드려요!

[mdn figure](https://developer.mozilla.org/ko/docs/Web/HTML/Element/figure)

```suggestion
              <img src="images/main/main_register.png" alt="인기 상품 이미지" />
```